silhouette/app/com/mohiva/play/silhouette/api/EventBus.scala

```diff
@@ -30,58 +30,58 @@ trait SilhouetteEvent
  *
  * @param identity The newly created identity.
  * @param request The request header for the associated request.
- * @param lang The lang associated for the current request.
+ * @param messages The messages for the current language.
  * @tparam I The type of the identity.
  */
-case class SignUpEvent[I <: Identity](identity: I, request: RequestHeader, lang: Lang) extends SilhouetteEvent
+case class SignUpEvent[I <: Identity](identity: I, request: RequestHeader, messages: Messages) extends SilhouetteEvent
```

Comment:
Is it necessary `Messages` as param.

Because the Subscriber can extends I18NSupport and get that.